docker-compose.yml

```diff
@@ -252,6 +267,8 @@ volumes:
     external: true
   sentry-symbolicator:
     external: true
+  sentry-geoip:
+    external: true
```

Comment:
~~Should this really be external? It is data we can easily recover so I'd make this a "local" volume.~~

We shouldn't even need this. I expect the downloader to put the database in a local folder on the host which we can set. So I'd add `geoip` directory to `.gitignore`, download the DB there and mount that directory to the containers.

Makes sense?

Reply:
Six of one, half-dozen of the other. I can do this, sure.

Reply:
Not sure what you mean by six of one. The reason I prefer a local folder mount is because it is easier to understand, easier to manage, and is more portable. What do you think?

Reply:
Sorry ... I mean I don't see a material difference between a bind mount and a volume mount here. Sure, it's a little more complicated to access the mmdb file from the host if it's in a volume, but it'll very rarely be needed from the host. It's not obvious to me how a bind mound improves manageability or portability.

Done in 7d2c36be2c0117f1959c494cfc0482195082aeff.

P.S. Why do we use the long syntax for read-only bind mounts, but the short syntax for read/write bind mounts and named volumes?

Reply:
>Sure, it's a little more complicated to access the mmdb file from the host if it's in a volume, but it'll very rarely be needed from the host. It's not obvious to me how a bind mound improves manageability or portability.

It is tricky to backup, move etc a named volume as Docker hides them away. This is especially an issue on non-Linux systems where things happen through a VM. I can easily imagine people copying their mmdb files around once they have a license once. External volumes also need managing from the install script (compose project scoped volumes don't have this issue but they are still opaque)

>P.S. Why do we use the long syntax for read-only bind mounts, but the short syntax for read/write bind mounts and named volumes?

Because we cannot specify the `readonly` flag in the short syntax? (or if we can, I don't know how to)

Reply:
Okay, okay. Bind mount is easier to work with here, I agree. 😁 

Re: syntax ... as I suspected. 😌 The short syntax does [allow](https://docs.docker.com/compose/compose-file/#short-syntax-3) for an optional mode at the end, like so:

`./foo:/foo:ro`

Reply:
>Okay, okay. Bind mount is easier to work with here, I agree. 😁

Hahaha, if you don't, we can still try. It is hard to be very objective about these things. 😁 

>Re: syntax ... as I suspected. 😌 The short syntax does allow for an optional mode at the end, like so:

Then I'm all in for making these short and consistent in a separate patch 🥳